### ISSUE TITLE
Inconsistency in logical expression alignment?

### ISSUE THREAD

**Issue by vonglan** [SAP/abap-cleaner#384]
I noticed this looks good

```
  IF    s_ivbeln[] IS NOT INITIAL
     OR s_ifkdat[] IS NOT INITIAL
     OR s_ierdat[] IS NOT INITIAL
```

but then why is the first line of this 

```
  IF ( s_vbeln[] IS NOT INITIAL AND s_vbeln[ 1 ]-low NS '*' )
     OR s_auart[]  IS NOT INITIAL
     OR s_ifkart[] IS NOT INITIAL
```

not indented the same way?

**Comment by jmgrassau** [comment]
Hi Edo, 

good point! This is always a bit tricky, but here's how alignment should look with the next release:

```ABAP
  IF     ( a = b )
     AND a = c.
  ENDIF.

  IF     a = b
     AND ( a = c ).
  ENDIF.

  IF        ( a = b OR a = 1 )
        AND a = c
     OR ( b = c OR c = 1 ).
  ENDIF.

  IF     (        ( a = b OR a = 1 )
              AND a = c
           OR (     b = c
                AND ( c = 1 ) ) )
     AND b = c.
  ENDIF.

  IF     (        ( a = b OR a = 1 )
              AND ( a = c )
           OR (     ( b = c )
                AND ( c = 1 ) ) )
     AND ( b = c ).
  ENDIF.

  IF        (    a = b
              OR c = d )
        AND a = c
     OR     a = d
        AND ( a = c OR b = d ).
  ENDIF.
```

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Edo, 

thanks again for posting this issue! This should now be fixed with version 1.12.0, which was just released.

Kind regards,
Jörg-Michael
